### PR TITLE
refactor(date-picker): [date-picker]  modify variable names and  styles for date-picker

### PR DESCRIPTION
**Pull request by MomoPoppy**
…es for date-picker

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **New Features**
	- Enhanced date and time picker functionality with improved state management and formatting options.
	- Introduced new CSS variables for better theming and styling consistency across components.
- **Bug Fixes**
	- Resolved issues with infinite loops when emitting database time.
- **Documentation**
	- Updated comments in CSS files to clarify the purpose of various variables.
- **Refactor**
	- Reorganized CSS variable naming conventions to improve clarity and maintainability.
	- Consolidated styles within various components for better structure and organization.

### CODE REVIEW

**Comment by MomoPoppy** [comment]
image
image
image
image
image

File: packages/vue/src/month-range/src/pc.vue [hunk 1]
```diff
@@ -83,18 +85,20 @@
...
-              :default-value="state.defaultValue"
-              :min-date="state.minDate"
-              :max-date="state.maxDate"
-              :range-state="state.rangeState"
-              :disabled-date="state.disabledDate"
-              @changerange="handleChangeRange"
-              @pick="handleRangePick"
-            >
-            </month-table>
+            <div class="tiny-date-range-picker__table">
```
Comment (zzcr) [inline packages/vue/src/month-range/src/pc.vue:88]: 改变了模板，需要评估下是否对saas主题造成影响？

**Comment by MomoPoppy** [comment]
image
image
image
image
image
image

pc模板的修改没有影响saas

File: packages/vue/src/month-range/src/pc.vue (hunk 1 above)
Comment (MomoPoppy) [reply packages/vue/src/month-range/src/pc.vue:88]: 已贴图，没有，就是在表格的外层加个div

**Review by zzcr — APPROVED** [review]

**Pull request merged**